Source/WebKit/Shared/RemoteLayerTree/RemoteLayerTreePropertyApplier.mm

```diff
@@ -267,8 +267,15 @@ static void updateCustomAppearance(CALayer *layer, GraphicsLayer::CustomAppearan
     if (properties.changedProperties & LayerChange::FiltersChanged)
         PlatformCAFilters::setFiltersOnLayer(layer, properties.filters ? *properties.filters : FilterOperations());
 
-    if (properties.changedProperties & LayerChange::AnimationsChanged)
+    if (properties.changedProperties & LayerChange::AnimationsChanged) {
+#if ENABLE(THREADED_ANIMATION_RESOLUTION)
+        if (layerTreeHost->threadedAnimationResolutionEnabled()) {
+            LOG_WITH_STREAM(Animations, stream << "RemoteLayerTreePropertyApplier::applyProperties() found " << properties.animationChanges.effects.size() << " effects");
+            layerTreeNode->setAcceleratedEffectsAndBaseValues(properties.animationChanges.effects, properties.animationChanges.baseValues, layerTreeHost);
```

Comment:
This probably doesn't matter, but since the two paths are reading different parts of `animationChanges`, maybe we should just call both?

Might help if the setting got changed between when the transaction was created and when it was received.

Reply:
I don't think it's worth bothering about this, at least now. For A/B testing I think it's reasonable to expect a page reload.